src/BlockchainTestsFiller/bcUncleHeaderValiditiy/futureUncleTimestampDifficultyDrop2Filler.json

```diff
@@ -88,6 +77,7 @@
 		"expectExceptionEIP150" : "InvalidStateRoot",
 		"expectExceptionEIP158" : "InvalidStateRoot",
                 "expectExceptionByzantium" : "InvalidStateRoot",
+                "expectExceptionConstantinople" : "InvalidStateRoot",
 		"blocknumber" : "3",
 		"blockHeader" : {
 			"stateRoot" : "3e9110dccee270e36e254d8e3b67cb18c12a9a0aa24065e24372a85856195c70",
```

Comment:
what you mean not introduced? you changed the third block to test nothing in this  test